.editorconfig

```diff
@@ -1,7 +1,7 @@
 root = true
 
 [*]
-end_of_line = lf
+end_of_line = crlf
```

Comment:
What? Why?

Reply:
It's just temporary since something was complaining on my machine and I'm too lazy to fix right now. `lf` is still going to be used

Reply:
Alright. Yeah, entire repo is forced to LF, so when you push it changes anyways. But you should fix your editor or local git. :D